pkg/pipeline/templates/terraformed.go.tmpl

```diff
@@ -24,13 +24,22 @@ func (tr *{{ .CRD.Kind }}) GetTerraformResourceIDField() string {
 
 // GetConnectionDetailsMapping for this {{ .CRD.Kind }}
 func (tr *{{ .CRD.Kind }}) GetConnectionDetailsMapping() map[string]string {
-  {{- if .SensitiveFields }}
-  return map[string]string{ {{range $k, $v := .SensitiveFields}}"{{ $k }}": "{{ $v}}", {{end}} }
+  {{- if .Sensitive.Fields }}
+  return map[string]string{ {{range $k, $v := .Sensitive.Fields}}"{{ $k }}": "{{ $v}}", {{end}} }
   {{- else }}
   return nil
   {{- end }}
 }
 
+// GetCustomConnectionKeys for this {{ .CRD.Kind }}
```

Comment:
Do you think we can follow an approach similar to https://github.com/crossplane-contrib/terrajet/pull/124 ?

Reply:
Yeah, definetely. I'll rebase on top of it so that it can go first.